README.rst

```diff
@@ -21,18 +21,18 @@ Development of this template itself
 We don't need to run inside a vm/docker ourselves, so to set it up and test
 it, just do the regular::
 
-  $ python3 bootstrap.py
-  $ bin/buildout
-  $ bin/test
+  $ pipenv install --dev
+  $ pipenv run nosetests
+
 
 The test, however, *does* use docker and docker-compose:
 
 - There's a test that checks if the template itself generates OK without
   errors.
 
 - There's a second test that uses the template-generated docker-compose setup
-  to run the ``bin/test`` of the generated django.
+  to run the ``nosetests`` of the generated django.
 
 We don't really need any python code ourselves, so our own ``setup.py``
-doesn't actually point at any code. But it is set up so that ``bin/test``
+doesn't actually point at any code. But it is set up so that ``nosetests``
```

Comment:
Misschien gelijk op py.test overstappen, dan sijpelt dat commando er gelijk in.

Reply:
Maak ik een aparte issue voor aan